tests/multiremote/test.js

```diff
@@ -23,6 +25,16 @@ describe('smoke test multiremote', () => {
             assert.strictEqual(results[0] >= 30, true, `First of [${results}] is less than 30`)
         })
 
+        it('should respect promises if command was added to single browser', () => {
+            browser.customCommandScenario(Object.keys(browser.instances).length)
+            global.browserA.addCommand('foobar', () => {
+                const title = global.browserA.getTitle()
+                return `Title: ${title}`
+            })
+            assert.strictEqual(global.browserA.foobar(), 'Title: Mock Page Title')
+            assert.equal(typeof global.browserB.foobbar, 'undefined')
```

Comment:
is it a typo in `foobbar`? two `b` letters

Reply:
typo, good catch 👍